### PR TITLE
[jeelink] add support for TX22 sensors and connected TX23 & TX26 sensors.

### PR DESCRIPTION
**Pull request by vbier**
This adds support for reading TX22 temperature and humidity sensors (as well as TX23 wind sensors and TX26 rain sensors wired to the TX22).

Signed-off-by: Volker Bier <volker.bier@web.de>

### CODE REVIEW

**Comment by vbier** [comment]
Okay, this seems to work as expected and is ready to be reviewed.

File: addons/binding/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/Tx22ReadingConverter.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+                return new Tx22Reading(sensorId, type, 0, temperature, humidity, batteryNew, batteryLow, rain,
+                        windDirection, windSpeed, windGust, pressure);
+            }
+        }
+
+        return null;
+    }
+
+    public static void main(String[] args) {
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/Tx22ReadingConverter.java]: This looks unexpected here

Comment (vbier) [reply addons/binding/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/Tx22ReadingConverter.java]: Has been removed.

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**